### ISSUE TITLE
Re-categorization: Implement new category on feature branch - specifically workplace-safety

### ISSUE THREAD

**Issue by debjudy** [18F/cv_faq#1046]
## Description
In order to improve content discovery, I want to update the "workplace-safety" category.

https://docs.google.com/spreadsheets/d/1j6do8VXXDEirwLoiGCZD4gNfJOfbE7BGB6uYBHfOE-I/edit#gid=595283323

## Acceptance criteria
- [x] Questions including singular and cross-categorized questions added/removed from category 
- [x] Questions ordered in category
- [x] Question level re-directs added/removed
- [x] [Re-direct spreadsheet](https://docs.google.com/spreadsheets/d/1F-uIkQmjju4tAUpympFM_9AuSFqHjEihOkQuotbaXwc/edit#gid=0) updated 

## Notes
- Cannot begin before completion of #1049 and #1046 
-  Each category will have its own issue so that we can track work progress and parse out activities to more than one person.

**Comment by debjudy** [comment]
Reviewed and closing
